### DISCUSSION TITLE
Specific questions regarding Pavo

### DISCUSSION THREAD

**Discussion by KellyRonald** [thomased/colRverse#9] [Q&A] [answered]
Hi all, 

I've had some specific questions asked from a reviewer that are detailed enough that I am not sure the answers; I thought I might ask the community here that could point me in the right direction. Some helpful info: we used PAVO to calculate achromatic/chromatic contrasts using the RNL model. I appreciate so much your help in advance. 

1) When the quantum catch is less than one, does PAVO have the flexibility to provide for the alternative q/(q+1) transformation to provide more reliable results?

2) What is used as the adapting spectrum in order to change from absolute quantum catches to relative quantum catches?

3) Is it possible within PAVO to select for ocular media transmittance from VS birds rather than UVS birds?

**Comment by thomased** [comment, marked as answer]
Hi Kelly, I've just emailed you, but copied here just fyi!
 
1. Yes, it does (qcatch = ‘Ei’ in vismodel() is equivalent to q/q+1). That’s making a slightly different assumption about how receptors are transforming their outputs, though, so it’s not quite as simple as it being a ‘correction’ of sorts for when catches are less than 1. At any rate, their comment only seems to be concerned with cases when quantum catches are less than one, which is unlikely unless they’ve been modelled under very dim-light conditions of if the stimuli are non-reflective (0) across large portions of the spectrum. So I wouldn’t worry too much.

2. It depends on what modelling was done, but there are two bits to this. One is that if you only ran the receptor-noise model, then you wouldn’t have used relative quantum catches at all. Separate to this is the adapting background question, which you can either do for the receptor-noise model (raw quantum catches), or a colourspace model (relative quantum catches). Long story short, the transformation for background adaptation doesn’t actually do anything in practical terms outside of some specific circumstances, and is controlled by the ‘vonkries’ argument in vismodel(). If you left the defaults though, you could say you used an ‘ideal’ (achromatic) adaptation background, and also note that the transformation has no effect on measures of stimulus-background distances. You can imagine it working like a white balance in a camera — different adaptation backgrounds will shuffle around the absolute position of points in a colourspace, but because you’re applying it to all points, their relative positions remain unchanged (i.e. the distance, or contrast, between stimuli and backgrounds).

3. Yep, you can specify and control various ocular media transmission details, oil droplets, etc. if you like, or there are some built in (see the help file for ?sensmodel()). If you used one of the built-in bird visual phenotypes you could just say that yes the sensitivities incorporate ocular transmission data, which can be modified if you like. The references are in that help file, for any more reading on what pavo has built-in.
